packages/cli/src/config.ts

```diff
@@ -0,0 +1,35 @@
+import { existsSync } from "fs-extra";
+import { getPackageJSON } from "./paths";
+import { writeFileSync } from "fs";
+
+export const MAILING_CONFIG_FILE = "./mailing.config.json";
+
+export function looksLikeTypescriptProject(): boolean {
+  if (existsSync("./tsconfig.json")) {
+    return true;
+  }
+
+  const pkg = getPackageJSON();
+  return !!(pkg.devDependencies?.typescript || pkg.dependencies?.typescript);
+}
+
+export function writeDefaultConfigFile(): void {
+  const fs = require("fs");
+  const prettier = require("prettier");
+
+  if (!existsSync(MAILING_CONFIG_FILE)) {
+    const configJsonString = prettier.format(
+      JSON.stringify({
+        typescript: looksLikeTypescriptProject(),
+        emailsDir: "./emails",
+      }),
+      { parser: "json", printWidth: 0 }
+    );
+
+    try {
+      writeFileSync(MAILING_CONFIG_FILE, configJsonString);
+    } catch (err) {
+      console.error(err);
```

Comment:
switch to [error fuction](https://github.com/sofn-xyz/mailing/blob/main/packages/cli/src/log.ts#L9)